src/libstore/binary-cache-store.cc

```diff
@@ -171,13 +171,13 @@ ref<const ValidPathInfo> BinaryCacheStore::addToStoreCommon(
     narInfo->fileHash = fileHash;
     narInfo->fileSize = fileSize;
     narInfo->url = "nar/" + narInfo->fileHash->to_string(HashFormat::Nix32, false) + ".nar"
-                   + (config.compression == "xz"      ? ".xz"
-                      : config.compression == "bzip2" ? ".bz2"
-                      : config.compression == "zstd"  ? ".zst"
-                      : config.compression == "lzip"  ? ".lzip"
-                      : config.compression == "lz4"   ? ".lz4"
-                      : config.compression == "br"    ? ".br"
-                                                      : "");
+                   + (config.compression == "xz"                                     ? ".xz"
+                      : config.compression == "bzip2"                                ? ".bz2"
+                      : config.compression == "zstd"                                 ? ".zst"
+                      : config.compression == "lzip"                                 ? ".lzip"
+                      : config.compression == "lz4"                                  ? ".lz4"
+                      : config.compression == "br" || config.compression == "brotli" ? ".br"
+                                                                                     : "");
```

Comment:
Side comment: `clang-format` wilding out with the ternary operator is something else :)